virttest/env_process.py

```diff
@@ -57,6 +58,37 @@
 QEMU_VERSION_RE = r"QEMU (?:PC )?emulator version\s([0-9]+\.[0-9]+\.[0-9]+)\s?\((.*?)\)"
 
 
+class VersionInterval:
```

Comment:
How about making a base abstraction of `Interval` and put it in `utils_numeric`?